features/security-monitoring.md

```diff
@@ -0,0 +1,28 @@
+# Security Monitoring
+The Private Packagist security monitoring feature analyzes the dependencies of your private packages for known security vulnerabilities.
+Packages get analyzed on every commit and every time a database reports new vulnerabilities.
+
+The following databases are used to analyze your packages:
+* [sensiolabs/security-advisories](https://github.com/FriendsOfPHP/security-advisories)
+
+### Configure monitoring settings
+The default branch of every private package gets automatically monitored if a composer.lock file is present.
+Additional branches can be configured on the package page.
+
+The feature can be disabled for individual packages on the package security page or for all packages
+on the organization’s security settings page.
+Organizations using the agency add-on also have the option to disable the feature for individual subrepositories.
+
+### Configure notifications
+Every user receives security notifications for all packages they have access to by default.
+They can unsubscribe from individual packages or from the feature all together if they do not with to receive any emails.
```

Comment:
Again don't speak of "the feature" it's vague and unclear what that refers to, the subscription feature?

"Users can unsubscribe either from individual packages, or from all security notifications, if they not wish to receive any email notifications."